File: Exercism Codes/JavaScript/custom-signs.js
export function buildSign(occasion, name) {
    return `Happy ${occasion} ${name}!`;
  }
  
  export function buildBirthdaySign(age) {
    return `Happy Birthday! What a ${age >= 50 ? "mature" : "young"} fellow you are.`
  }
  
  export function graduationFor(name, year) {
    return `Congratulations ${name}!\nClass of ${year}`;
  }
  
  export function costOf(sign, currency) {
    return `Your sign costs ${(sign.length * 2 + 20).toFixed(2)} ${currency}.`;
  }

File: Exercism Codes/Swift/CustomSigns.swift
let birthday: String = "Birthday";
let valentine: String = "Valentine's Day";
let anniversary: String = "Anniversary";
let space: Character = " ";
let exclamation: Character = "!";

func buildSign(for occasion: String, name: String) -> String {
  return "Happy" + String(space) + occasion + String(space) + name + String(exclamation);
}

func graduationFor(name: String, year: Int) -> String {
  return "Congratulations \(name)!\nClass of \(year)";
}

func costOf(sign: String) -> Int {
  return sign.count * 2 + 20;
}
